### PR TITLE
rosdep: nixos: Add transforms3d packages

### PR DESCRIPTION
**Pull request by hacker1024**

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -4276,6 +4276,7 @@ python-transforms3d-pip:
   debian:
     pip:
       packages: [transforms3d]
+  nixos: [python3Packages.transforms3d]
```
Comment (clalancette) [inline rosdep/python.yaml:4279]: There's no real reason to add to this one; this is implicitly Python 2, which is no longer supported.

```suggestion
```

File: rosdep/python.yaml [hunk 2]
```diff
@@ -9113,6 +9114,7 @@ python3-transforms3d:
...
   fedora: [python3-transforms3d]
```
Comment (clalancette) [inline rosdep/python.yaml:9118]: This needs to be in alphabetical order:

```suggestion
  fedora: [python3-transforms3d]
  nixos: [python3Packages.transforms3d]
```

**Review by clalancette — CHANGES_REQUESTED** [review]

**Comment by tfoote** [comment]
I merged the fix for the formatting but I don't have a link to validate the package. Please fill out the description with more information so we can validate the package linked is correct.

**Review by sloretz — APPROVED** [review]
https://search.nixos.org/packages?channel=23.11&show=python311Packages.transforms3d&from=0&size=50&sort=relevance&type=packages&query=transforms3d

**Review by clalancette — APPROVED** [review]

**Pull request merged**